maxim/layers.py

```diff
@@ -76,28 +100,60 @@ def get_config(self):
 
 
 @tf.keras.utils.register_keras_serializable("maxim")
-class Resizing(layers.Layer):
```

Comment:
What is the need to segregate this to Up and Down?

Reply:
I found it easier to read, but indeed it adds a chunk of code. Reformatting to use a single layer only.

Reply:
If it's easier to read, I would consider adding an elaborate comment in the script so that readers are aware.

Reply:
I think now is better (with a single resizing layer).